go/QnAMaker/rest/create-kb.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+    "bytes"
+    "encoding/json"
+    "fmt"
+    "io/ioutil"
+    "net/http"
+    "strconv"
+    "time"
+)
+
+var host string = "https://{your-resource-name}.api.cognitive.microsoft.com"
```

Comment:
URL format is currently the following .
https://{your-resource-name}.cognitiveservices.azure.com
Please change across all languages code samples.